main.go

```diff
@@ -73,6 +73,24 @@ var (
 
 	runningTests = false
 
+	help               = kingpin.CommandLine.HelpFlag.Short('h')
+	linter             = kingpin.Flag("lint", "enable linter check").Bool()
```

Comment:
Note that you want `tyk lint`, not `tyk --lint`.

If kingpin doesn't handle this, you should use something like `flag.Args()` (the kingpin version of that).

Reply:
Easily fixed I think